fastlane/Appfile

```diff
@@ -1,9 +1,11 @@
-app_identifier("com.chat.expensify.chat") # The bundle identifier of your app
-apple_id("ios@expensify.com") # Your Apple email address
-
-itc_team_id("152696") # App Store Connect Team ID
-team_id("368M544MTT") # Developer Portal Team ID
+# See https://docs.fastlane.tools/advanced/Appfile/
+apple_id("ios@expensify.com")
+itc_team_id("152696")
+team_id("368M544MTT")
 
 for_lane :build_hybrid, :build_unsigned_hybrid, :upload_testflight_hybrid do
+  # For HybridApp build tasks, use the OldApp bundle identifier
   app_identifier("com.expensify.expensifylite")
+else
+  app_identifier("com.expensify.expensify")
```

Comment:
Co-pilot suggested the wrong name here, didn't notice 😞

Reply:
Oh for some reason I thought this was correct when I reviewed too 🤦